[2019-05-28 19:47:58] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /index"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:47)"} []
[2019-05-28 21:17:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /backend/"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /backend/\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:47)"} []
[2019-05-28 21:18:14] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /backend/"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /backend/\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:47)"} []
[2019-05-30 21:26:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev-php/pruebas"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev-php/pruebas\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:68)"} []
[2019-05-30 21:27:52] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev-php/pruebas"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev-php/pruebas\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:68)"} []
[2019-05-30 21:28:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev-php/pruebas"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev-php/pruebas\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:68)"} []
[2019-05-30 21:31:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev-php/pruebas"" at C:\wamp\www\curso-backfront\symfony\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev-php/pruebas\" at C:\\wamp\\www\\curso-backfront\\symfony\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:128, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:59)"} []
[2019-05-30 21:31:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev-php/pruebas"" at C:\wamp\www\curso-backfront\symfony\var\cache\prod\classes.php line 4263 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev-php/pruebas\" at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\classes.php:4263, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\wamp\\www\\curso-backfront\\symfony\\var\\cache\\prod\\appProdProjectContainerUrlMatcher.php:59)"} []
[2019-05-31 06:30:19] request.INFO: Matched route "default_pruebas". {"route":"default_pruebas","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::pruebaAction","_route":"default_pruebas"},"request_uri":"http://localhost:8080/curso_backfront/symfony/web/pruebas","method":"GET"} []
[2019-05-31 06:30:19] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-05-31 06:30:19] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront'" at C:\wamp64\www\curso_backfront\symfony\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
[2019-05-31 06:30:24] request.INFO: Matched route "default_pruebas". {"route":"default_pruebas","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::pruebaAction","_route":"default_pruebas"},"request_uri":"http://localhost:8080/curso_backfront/symfony/web/pruebas","method":"GET"} []
[2019-05-31 06:30:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-05-31 06:30:25] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront'" at C:\wamp64\www\curso_backfront\symfony\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
[2019-05-31 06:31:11] request.INFO: Matched route "default_pruebas". {"route":"default_pruebas","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::pruebaAction","_route":"default_pruebas"},"request_uri":"http://localhost:8080/curso_backfront/symfony/web/pruebas","method":"GET"} []
[2019-05-31 06:31:11] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-05-31 06:31:11] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront'" at C:\wamp64\www\curso_backfront\symfony\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[HY000] [1049] Unknown database 'curso_backfront' at C:\\wamp64\\www\\curso_backfront\\symfony\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
